pkg/pillar/cmd/nim/nim.go

```diff
@@ -253,6 +253,11 @@ func (n *nim) run(ctx context.Context) (err error) {
 	defer close(done)
 	netEvents := n.networkMonitor.WatchEvents(ctx, agentName)
 
+	// Time limit to obtain some network config.
+	// If it runs out and we still do not have any config,
+	// lastresort will be enabled unconditionally.
```

Comment:
unconditionally? If the user has set network.fallback.any.eth to disabled then even after a minute we shouldn't enable lastresort. I don't know if this is just in the wording in the comment or if the code does the same.

Reply:
Sorry, my confusion.
But it makes sense to add to the comment "this is to handle the case that for whatever reason the device has lost /persist/status/nim/DevicePortConfigList".

Reply:
Done (added comment)